### DISCUSSION TITLE
Error when trying to add new object to Netbox

### DISCUSSION THREAD

**Discussion by buzzlightyear2k** [netbox-community/netbox#9908] [Q&A]
Im trying to add Load Balancer and Virtual Endpoint to netbox.When im adding specific CP Endpoint to Load Balancer i cannot select any Load Balancer.
![image](https://user-images.githubusercontent.com/86591649/182522739-b27f1242-ade4-4f88-b396-48d89353117d.png)
My load balancer on dtb : 
![image](https://user-images.githubusercontent.com/86591649/182522782-8622d10a-7275-41c7-8ed3-ce5aca8341fc.png)

**Comment by jeremystretch** [comment]
You appear to be using some forked version of NetBox. You'll have to contact its maintainer for assistance.
